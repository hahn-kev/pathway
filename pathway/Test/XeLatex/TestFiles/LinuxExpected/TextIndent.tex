\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\t="Times New Roman" at 15pt
\font\verset="Times New Roman" at 15pt
\font\salvationt="Times New Roman" at 15pt

\pagestyle{fancy} 

\t{You and all the people of Israel: It is by the name of Jesus Christ of Nazareth, whom you crucified but whom God raised from the dead, that this man stands before you healed. }\verset{11}\t{ He is " 'the stone you builders rejected, which has become the capstone.[a]'[b] }\verset{12}\salvationt{  Salvation is found in no one else, for there is no other name under heaven given to men by which we must be saved." }

\end{document}
