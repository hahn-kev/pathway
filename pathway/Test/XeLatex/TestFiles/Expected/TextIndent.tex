\documentclass[a4paper]{article} 
\usepackage{texthanglist}
\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage[margin=2cm,includeheadfoot]{geometry}

\begin{document} 
\pagestyle{plain} 
\font\salvationt="Times New Roman" at 15pt
\font\verset="Times New Roman" at 15pt
\font\t="Times New Roman" at 15pt
\pagestyle{fancy} 

\t{You and all the people of Israel: It is by the name of Jesus Christ of Nazareth, whom you crucified but whom God raised from the dead, that this man stands before you healed. }\verset{11}\t{He is " 'the stone you builders rejected, which has become the capstone.[a]'[b] }\verset{12}\salvationt{ Salvation is found in no one else, for there is no other name under heaven given to men by which we must be saved." }

\end{document}
